include/zim/writer/creator.h

```diff
@@ -74,15 +74,24 @@ namespace zim
         Creator& configCompression(CompressionType comptype);
 
         /**
-         * Set the minimum size of the cluster.
+         * Set the size of the crated clusters.
          *
-         * Creator will try to create cluster with this minimum size
-         * (uncompressed size).
+         * The creator will try to create cluster with (uncompressed) size
+         * the closer (but smaller) to requestSize.
+         * If not possible, (one content bigger than requestSize),
+         * the cluster will have only one (the) content
+         * and will be bigger than requestSize.
          *
-         * @param size The minimum size of a cluster.
+         * Be carefull with this value.
+         * Bigger value means more content put together, so a better compression ratio.
+         * But it means also that more decompression has to be made when reading a blob.
+         * If you don't know which value to put, don't use this method and let libzim
+         * use the default value.
+         *
+         * @param requestSize The minimum size of a cluster (in byte).
```

Comment:
"minimum" is not a good adjective here. It is simply wrong, since most clusters will have size smaller than this value. Change to either "maximum", "desired" or "target".

Reply:
I've missed this when I've rewrite the docstring.